cves/kernel/CVE-2015-2041.yml

```diff
@@ -147,10 +144,18 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: 
+    A flaw was found in the method that the linux kernel handles userspace configuration 
+    of  of the the Linux kernel's logical link control (LLC) implementation  system settings. 
+    The incorrect handling allowed a trusted user to set multiple LLC syscalls with specially formatted data. 
+    Reading from these files also returned data from other sysctl settings that would be exposed via the 
```

Comment:
Again this does not explain how the exploit was found.

Reply:
Yep. Will fix this soon. Thanks!